users/views.py

```diff
@@ -0,0 +1,21 @@
+from .forms import CustomUserCreationForm
+from django.urls import reverse_lazy
+from django.views import generic
+from django.shortcuts import render
+from .models import CustomUser
+
+
+class SignUpView(generic.CreateView):
+    form_class = CustomUserCreationForm
+    success_url = reverse_lazy('login')
```

Comment:
It's good to show the user that the confirmation email was sent so that he know what to do next.